src/views/home/NewCommunity.vue

```diff
@@ -0,0 +1,209 @@
+                v-model="website" required>
+              </div>
+            </div>
+          </div>
+
+          <div class="row">
+            <div class="col-md-6">
+              <div class="form-group">
+                <label for="type" class="form-label">Type *</label>
+                <select class="form-control" id="type"
+                v-model="content_types" required>
+                  <option>{{CONTENT_TYPES.GAME}}</option>
+                  <option>{{CONTENT_TYPES.MOVIE}}</option>
+                  <option>{{CONTENT_TYPES.BOOK}}</option>
+                  <option>{{CONTENT_TYPES.SERIES}}</option>
+                </select>
+              </div>
+            </div>
+            <div class="col-md-6">
+              <div class="form-group">
+                <label for="url">Image path * <small class="text-muted"> Please include http://</small></label>
+                <input type="url" class="form-control" id="url" placeholder="Image path"
+                 v-model="image_path" required>
+              </div>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-md-12 text-right">
+              <button type="reset" class="btn btn-link mx-2 reset">Reset</button>
+              <button type="submit" class="btn btn-success submit">Submit</button>
+            </div>
+          </div>
+        </form>
+    </div>
+        <div class="col-md-3 info">
+          <h6>About Moderating Groups</h6>
+          <ul>
+            <li>Only moderators can edit group settings.</li>
+            <li>You can add and remove moderators on the members page.</li>
+            <li>You can block users from joining this group on the block/unblock users page.</li>
+            <li>For more information about being a moderator, read the moderator guidelines.</li>
+          </ul>
+        </div>
+    </div>
+  </div>
+</template>
+<script>
+import axios from 'axios';
+import swal from 'sweetalert';
+import $ from 'jquery';
+
+import Multiselect from '@/components/Multiselect.vue';
+
+import { CONTENT_TYPES } from '@/constants';
+
+export default {
+  name: 'NewCommunity',
+  components: {
+    Multiselect,
+  },
+  data() {
+    return {
+      organizers: [],
+      members: [],
+      name: '',
+      image_path: '',
+      content_types: '',
+      description: '',
+      tags: [],
+      website: '',
+      rules: '',
+      CONTENT_TYPES,
+      dataItem: [
+        { name: 'Vue.js', code: 'vu' },
+        { name: 'Javascript', code: 'js' },
+        { name: 'Open Source', code: 'os' },
+      ],
+      community_id: '',
+    };
+  },
+  methods: {
+    submitForm() {
+      axios.post('/api/communities', {
+        name: this.name,
+        image_path: this.image_path,
+        content_types: this.content_types,
+        description: this.description,
+        website: this.website,
+        rules: this.rules,
+        tags: this.tags.name,
```

Comment:
`this.tags` bir dizi olduğu için `name` adında bir key'i yoktur. Dizinin içindeki her elemanın name değerine ulaşmak ve bunları bir dizi içinde toplamak istiyorsan `map` fonksiyonunu kullanabilirsin